deploy/helm/rocketride/templates/chroma-deployment.yaml

```
@@ -0,0 +1,94 @@
+    - port: {{ .Values.chroma.service.port }}
+      targetPort: chroma
+      protocol: TCP
+      name: http
+  selector:
+    {{- include "rocketride.selectorLabels" . | nindent 4 }}
+    app.kubernetes.io/component: chroma
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "rocketride.fullname" . }}-chroma-data
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)